numpy/random/_generator.pyx

```diff
@@ -5022,13 +5022,13 @@ def default_rng(seed=None):
     is instantiated. This function does not manage a default global instance.
 
     See :ref:`seeding_and_entropy` for more information about seeding.
-    
+
     Examples
     --------
     `default_rng` is the recommended constructor for the random number class
     `Generator`. Here are several ways we can construct a random 
     number generator using `default_rng` and the `Generator` class. 
-    
+
```

Comment:
I don't know if we want to have whitespace trimming in unrelated files.

Reply:
I'm happy to revert the change if needed. I just happened to notice it and thought it would be okay to fix. I'll move most of the changes to a separate PR in a moment as suggested above so that this PR can be limited to be related to just "enabling" the examples.